app/src/main/java/ch/epfl/sdp/drone3d/ui/mission/ItineraryCreateActivity.kt

```diff
@@ -316,8 +328,22 @@ class ItineraryCreateActivity : BaseMapActivity(), OnMapReadyCallback,
         } else {
             goToSaveActivity()
         }
+
+
+    }
+
+    /**
+     * Test if the size of the area is with the accepted size
+     */
+    private fun isSizeWithinLimit(): Boolean {
+        return if (areaBuilder.isComplete()) {
+            areaBuilder.getAreaSize() < MAXIMUM_AREA_SIZE
+        } else {
+            false
+        }
```

Comment:
You could have wrote it using && due to the short-circuit evaluation
```suggestion
        return areaBuilder.isComplete() && areaBuilder.getAreaSize() < MAXIMUM_AREA_SIZE
```

Reply:
Nice one !